web/src/styles/pixel-skill-card.css

```
@@ -0,0 +1,595 @@
+  padding: 7px 10px 5px;
+  background: var(--psc-frame-a);
+  border-bottom: var(--psc-px) solid var(--psc-frame-edge);
+}
+.pixel-skill-card__name {
+  font-family: "Press Start 2P", "VT323", monospace;
+  font-size: 13px;
+  line-height: 1.25;
+  letter-spacing: 0.5px;
+  color: var(--psc-ink);
+  text-shadow: 1px 1px 0 rgba(255, 255, 255, 0.45);
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)